infra/charts/turing/values.yaml

```diff
@@ -263,8 +276,10 @@ mlp:
     # -- Name of MLP's Postgresql deployment
     nameOverride: postgresql-mlp
     containerPorts:
-      postgresql: 5432
-
+      postgresql: 5432 
+  environmentConfigSecret:
+    name: "" # set by mlp if needed
```

Comment:
Is `name` expected to be set to the Secret name only if the secret containing the `k8s cluster config` already exists?

Reply:
Will modify the comment to make it more general, but yes, this `name` points to a secret that contains the `K8sConfig`.

Reply:
Have added some comments.